apps/web/src/messages/employeeProfileMessages.ts

```diff
@@ -6,11 +6,6 @@ const messages = defineMessages({
     id: "tSvuK1",
     description: "Label for a employee profile about you field",
   },
-  careerGoals: {
-    defaultMessage: "Your career goals",
-    id: "NlcepR",
-    description: "Label for a employee profile career goals field",
-  },
   learningGoals: {
```

Comment:
I think you need to clear the translation associated with this to get the check to pass
`pnpm run check-intl --force`

Reply:
Thanks, got it.
[clean intl](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12772/commits/d5d19863a5be5e45fc1a7c77e96ccfc68b400969)